src/components/Footer/Footer.js

```diff
@@ -1,6 +1,55 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Footer.scss';
 
 export default function Footer() {
-  return <footer className="footer">Footer</footer>;
+  return (
+    <footer className="footer">
+      <ul className="listFooter">
+        <li>
+          <img src="/images/common/img_logo_w.png" alt="Plait" width="86px" />
+          <p>All rights reserved.</p>
+          {/* <span>회사 브랜드 이름, 로고, All rights reserved.</span> */}
+        </li>
+        <li>
+          <strong className="listTitle">회사정보</strong>
+          <p className="listDescription">
+            <span className="descInfo">㈜서윤식물 </span>
+            <span className="descInfo">CEO - 강서윤 </span>
+            <span className="descInfo">E-mail - hiImSeoyunKang@Plait.com </span>
+            <span className="descInfo">N - 892-75-03231 </span>
+            <span className="descInfo">Phone - 02 - 5024 -7859 </span>
+            <span className="descInfo">
+              Address - 서울특별시 강남구 테헤란로 427, 11F{' '}
+            </span>
+            <span className="descInfo">
+              통신판매업 신고 제 2023-서울-0001호
+            </span>
```

Comment:
```suggestion
           {INFO_LIST.map(text => {
              return <span key={text} className="descInfo">{text}</span>
           })}
```

이 부분은 아래 상수데이터를 만들어서 구현하는게 조금 더 깔끔하고
추후 내용이 추가 되더라도 유지보수 하기 좋습니다!